.github/workflows/build.yml

```diff
@@ -118,12 +120,14 @@ jobs:
         password: ${{ secrets.DOCKERHUB_TOKEN }}
     - name: Rebuild and push ${{ matrix.platforms }} docker images
       if: github.event_name != 'pull_request'
-      run: |
-        docker buildx build ${{ matrix.dockerContext }} \
-            --no-cache \
-            -t "${{ steps.create_docker_tag.outputs.TAG }}" \
-            --build-arg BASE_IMAGE_TAG=${{ matrix.baseImageTag }} \
-            --build-arg SBT_VERSION=${{ steps.get_sbt_version.outputs.VERSION }} \
-            --build-arg SCALA_VERSION=${{ matrix.scalaVersion }} \
-            --platform ${{ matrix.platforms }} \
-            --push
+      uses: docker/build-push-action@v4
+      with:
+        context: ${{ matrix.dockerContext }}
+        no-cache: true
```

Comment:
I wonder if we actually need this here or in the previous step. Any idea @kijuky ? I don't remember why I added this. For a proper image reuse from the previous step we need this cache.
We might also want a `pull: true` to make sure we always get the latest images?

Reply:
This is just a guess, but I thought it was because the build settings (ex: platforms) were different between the steps before and after. If those steps have the same build settings, I think you can use caching. I will rewrite it.

Reply:
https://github.com/docker/buildx/issues/59